### PR TITLE
Improve comments and user messages.

### PR DESCRIPTION
**Pull request by nwieters**

### CODE REVIEW

File: tar_binary_restarts/tar_binary_restarts.py [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
-import os
 import subprocess
 import time
-import pdb
 from esm_parser import user_note
 
 def tar_binary_restarts(config, test_config=None):
     """
-    Tar binary restart files.
+    Tar and compress binary restart files.
```
Comment (mandresm) [inline tar_binary_restarts/tar_binary_restarts.py]: Can you add here that this is specific to FESOM-2?

**Review by mandresm — APPROVED** [review]
Great! Thanks for the hard work!

Only one comment for the docstrings.

File: tar_binary_restarts/tar_binary_restarts.py (hunk 1 above)
Comment (nwieters) [reply tar_binary_restarts/tar_binary_restarts.py]: Done

**Comment by nwieters** [comment]
Thanks for the review.
It was my pleasure.

**Pull request merged**